jhub_apps/templates/style.css

```diff
@@ -422,10 +422,6 @@ a:focus {
     outline-color: var(--secondary-color) !important;
 }
 
-.MuiButton-textPrimary {
-    color: var(--primary-color) !important;
-}
-
```

Comment:
This  should not be removed, it affects other things.

Reply:
Returned styles